manage/sawtooth_manage/docker.py

```diff
@@ -318,3 +304,40 @@ def is_running(self, node_name):
                 return entry.status.startswith("Up")
         return False
 
+
+# docker image checking -- these need to be kept up to date
+
+def _get_unbuilt_images(processors):
+    sawtooth_processors = [
+        'sawtooth-' + processor
+        for processor in processors + ['validator']
+    ]
+    built_ins = _built_in_processor_types()
+    built_images = _get_built_images()
+
+    unbuilt = [image for image in sawtooth_processors
+               if image not in built_images and image in built_ins]
+
+    return unbuilt
+
+def _check_invalid_processors(processors):
+    sawtooth_processors = [
+        'sawtooth-' + processor
+        for processor in processors + ['validator']
+    ]
+    built_ins = _built_in_processor_types()
+    built_images = _get_built_images()
+
+    invalid = [image for image in sawtooth_processors
+               if image not in built_images and image not in built_ins]
+
+    return invalid
+
+def _get_built_images():
+    docker_img_cmd = ['docker', 'images', '--format', '{{.Repository}}']
+    return subprocess.check_output(docker_img_cmd).decode().split('\n')
+
+def _built_in_processor_types():
+    image_data_dir = os.path.join(
+        os.path.dirname(__file__), '..', '..', 'docker')
```

Comment:
This is no longer a valid way to get the built-in types as these docker files will not be installed.